.gitignore

```diff
@@ -58,3 +58,7 @@ client/.eslintcache
 
 # E2E Testing
 ignoreE2E*
+test.csv
+test_genesets.csv
+test-backups/
+test_genesets-backups/
```

Comment:
I believe these names are unique to Colin's personal workflow.  Why add to the repo?